src/software/new_geom/point.cpp

```diff
@@ -118,3 +118,23 @@ bool operator!=(const Point &p, const Point &q)
 {
     return !(p == q);
 }
+
+Point Point::getAverageOfPoints(std::vector<Point> points)
```

Comment:
Given that this function isn't specific to a single point, I think it probably belongs in `util.cpp`? @MathewMacDougall for your opinion.

Reply:
Deleted getAveragePoints() and new use getPointMean()